release_notes/48-release-notes.adoc

```diff
@@ -426,6 +427,12 @@ This release also addresses the following security vulnerabilities:
 
 *Release date* {ga-date-483}
 
+*Known issues*
+
+Central fails to start when using {product-title-short} with Federal Information Processing Standards (FIPS)-enabled clusters. Do not upgrade to release 4.8.3 if you have FIPS-enabled clusters. You must upgrade to release 4.8.4 or later.
```

Comment:
Suggestion to mention Central because we believe Secured Clusters aren't affected. This is optional, though.

```suggestion
Central fails to start when using {product-title-short} with Federal Information Processing Standards (FIPS)-enabled clusters. Do not upgrade Central to release 4.8.3 if you have FIPS-enabled clusters. You must upgrade to release 4.8.4 or later.
```